packages/client-python/src/rocketride/_engine/resolver.py

```
@@ -0,0 +1,94 @@
+
+    candidates = []
+    for release in releases:
+        tag = release.get('tag_name', '')
+        match = _TAG_PATTERN.match(tag)
+        if not match:
+            continue
+        version_str = match.group(1)
+        try:
+            version = Version(version_str)
+        except Exception:
+            continue
```

Ruff (0.15.7)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)